cmp/tab/TabSwitcher.js

```diff
@@ -39,7 +39,8 @@ export class TabSwitcher extends Component {
             onChange: this.onTabChange,
             large: !vertical,
             selectedTabId: selectedId,
-            items: children.map(({id, name}) => tab({id, title: name}))
+            items: children.map(({id, name}) => tab({id, title: name})),
+            ...this.props
```

Comment:
This we should pull the model off of the props to avoid passing that through to the underlying Blueprint tabs component (in case it decided to complain about an unknown prop or, worse, declare its own model prop with a different meaning).

Reply:
Nice catch!